docs/GLOSSARY.md

```diff
@@ -0,0 +1,9 @@
+# Glossary
+
+Defined terms in the documentation for Terraform Example Foundation are capitalized and have
+specific meaning within the domain of knowledge.
+
+## Terraform Service Account
```

Comment:
Looks good to start, some additional terms  here could be `seed-project`, `foundation-pipeline`, `app-infra-pipeline`

Reply:
Done